userspace/libscap/scap_savefile.h

```diff
@@ -61,6 +61,13 @@ typedef struct _section_header_block
 	uint64_t section_length;
 }section_header_block;
 
+// NB:
+// Starting from scap version 1.2, block versions will no longer be changed.
+// New block fields must be appended and, instead of using the version, the
+// lengths of the sub blocks will be used to differentiate between versions.
+// For more infomation, look at the comments inside the various scap_read_*
+// functions.
+
 ///////////////////////////////////////////////////////////////////////////////
 // MACHINE INFO BLOCK
 ///////////////////////////////////////////////////////////////////////////////
```

Comment:
Would it make sense to add the sublen also to this other block, since it's the only one missing? Completely fine if you say "no".

Reply:
I skipped the machine info block because, unlike the other ones, there's already some relatively big space reserved for additional fields (32 bytes - see https://github.com/draios/sysdig/blob/dev/userspace/libscap/scap.h#L300).